app/controllers/nsm/steps/view_claim_controller.rb

```diff
@@ -18,53 +18,89 @@ def item
         render report_params[:item_type]
       end
 
-      def work_items
+      def claimed_work_items
+        @report = CheckAnswers::ReadOnlyReport.new(current_application)
+        @claim = current_application
         @work_items = Sorters::WorkItemsSorter.call(current_application.work_items, @sort_by, @sort_direction)
 
-        render prefixed_view_for('work_items')
+        render 'nsm/steps/view_claim/show'
```

Comment:
I _think_ this can just be `render :show`. The "nsm/steps/view_claim" stuff is implied by the namespace/name of the controller, and once you've got rid of the slashes the string is simple enough it can be replaced by a symbol.

Reply:
I'm sure I did and it picked up another but I'll try again